### PR TITLE
Add document on installing NVIDIA CUDA Toolkit

### PR DESCRIPTION
**Pull request by puneetse**
This PR adds a new tutorial for installing the NVIDIA CUDA Toolkit.

Closes https://github.com/clearlinux/clear-linux-documentation/issues/464

### CODE REVIEW

**Comment by puneetse** [comment]
@lebensterben FYI.

File: source/clear-linux/tutorials/nvidia-cuda.rst [hunk 1]
```diff
@@ -0,0 +1,275 @@
+.. _nvidia:
+
+Install NVIDIA CUDA Toolkit 
+###########################
+
+NVIDIA is a manufacture of graphics processing units (GPU), also known as
```
Comment (bwarden) [inline source/clear-linux/tutorials/nvidia-cuda.rst]:
```suggestion
NVIDIA is a manufacturer of graphics processing units (GPU), also known as
```

File: source/clear-linux/tutorials/nvidia-cuda.rst [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+installed.
+
+
+#. Configure the dynamic linker to look for and cache shared libraries under
+   :file:`/opt/cuda/lib64` where the NVIDIA installer will place libraries. 
+
+   .. code-block:: bash
+      
+      echo "include /etc/ld.so.conf.d/*.conf" |  sudo tee --append /etc/ld.so.conf
+      sudo mkdir /etc/ld.so.conf.d
```
Comment (bwarden) [inline source/clear-linux/tutorials/nvidia-cuda.rst]: I'd swap these two lines (do the mkdir first) as a matter of pragmatism, plus add "-p" to mkdir.

**Comment by puneetse** [comment]
@mvincerx ready for final review and merge

File: source/clear-linux/tutorials/tutorials.rst [hunk 3]
```diff
@@ -24,5 +24,7 @@ Explore our tutorials to discover what you can do with |CL|!
    kubernetes/kubernetes*
    yubikey-u2f
    nvidia
+   nvidia-cuda
+   dars
```
Comment (mvincerx) [inline source/clear-linux/tutorials/tutorials.rst]: Remove reference to `dars`, please.  It's been moved here: 
https://github.com/clearlinux/clear-linux-documentation/tree/master/source/clear-linux/guides/stacks

File: source/clear-linux/tutorials/nvidia-cuda.rst [hunk 4]
```diff
@@ -0,0 +1,276 @@
+.. _nvidia:
```
Comment (mvincerx) [inline source/clear-linux/tutorials/nvidia-cuda.rst]: Update the filename to: `nvidia-cuda` here for a clean build in the docs.

**Comment by puneetse** [comment]
> Thanks for this new doc. We need a couple revisions for a clean build.
Done.

**Pull request merged**